server/src/test/java/org/jetbrains/bsp/bazel/server/bloop/ReGlobberTest.kt

```diff
@@ -0,0 +1,60 @@
+package org.jetbrains.bsp.bazel.server.bloop
+
+import bloop.config.Config
+import io.kotest.matchers.collections.shouldBeEmpty
+import io.kotest.matchers.shouldBe
+import org.jetbrains.bsp.bazel.server.sync.model.SourceSet
+import org.junit.jupiter.api.Test
+import java.nio.file.Paths
+import io.vavr.collection.HashSet
+import org.jetbrains.bsp.bazel.server.bloop.ScalaInterop.emptyList
+import org.jetbrains.bsp.bazel.server.bloop.ScalaInterop.toList
+import scala.jdk.javaapi.CollectionConverters
+
+class ReGlobberTest {
```

Comment:
how about the case when user specifies files by name? of example only one in the directory?

Reply:
right now it'll still reglob anything, even if there the sources don't represent the entire directory.

Reply:
is it an expected behavior / what is the reason?

Reply:
> IJ doesn't support more complicated globs

we'll fix it soon